java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -885,6 +887,46 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
 
         Object[] result = {
             "0-1", // xadd(streamKey1, Map.of("field1", "value1"), ... .id("0-1").build());
+            Map.of( // xinfoStream(streamKey1)
+                    "radix-tree-keys",
+                    1L,
+                    "radix-tree-nodes",
+                    2L,
+                    "entries-added",
+                    1L,
+                    "length",
+                    1L,
+                    "groups",
+                    0L,
+                    "max-deleted-entry-id",
```

Comment:
A handful of the fields returned by XINFO STREAM were added in 7.0.0 or 7.2.0 so I think the test for this will fail with 6.2.14. Info about which fields were added when is in the redis docs in the "history" section. We probably need to do a custom transaction test that only asserts against fields that exist in all versions like I did for Python.